src/tools/rustfmt/tests/source/frontmatter_compact.rs

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env cargo
+---identifier
+[dependencies]
+regex = "1"
+---
+#![feature(frontmatter)]
+
+fn main() {}
```

Comment:
Should this check that the file as a whole is being formatted? I think rustfmt sometimes bails out on formatting just from the presence somewhere in the file of syntax... not sure if that's happening here or not (or quite what the test format here looks like).

It probably makes sense to try to get rustfmt team review (do you have someone to re-assign already?). Otherwise, if we think that's not feasible, happy to just rubberstamp.

Reply:
@ytmimi (since you were assigned), any patterns in rustfmt for "prove that formatting hasn't bailed out early"?

Reply:
just misformat the function definitions in the files under `/source`

Reply:
Does that means that no change is then needed for this or should I still change `main`s formatting?

Reply:
dealer's choice, it wouldn't hurt to tweak the signature but also not necessary. i'd be fine with this going ahead as-is